<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompanyWindow</class>
 <widget class="QMainWindow" name="CompanyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employee Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="showCombobox">
        <item>
         <property name="text">
          <string>Show All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Show Others Only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Show Secretaries Only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Show Managers Only</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="employeesList">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="fireEveryoneButton">
          <property name="text">
           <string>Fire Everyone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hireSomeoneButton">
          <property name="text">
           <string>Hire Someone New</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEditor">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="salaryLabel">
          <property name="text">
           <string>Salary</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="salaryEditor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="bonusLabel">
          <property name="text">
           <string>Bonus %</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="bonusEditor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QRadioButton" name="employeeRButton">
          <property name="text">
           <string>Employee</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">employeeTypeRadioButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="secretaryRButton">
          <property name="text">
           <string>Secretary</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">employeeTypeRadioButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="managerRButton">
          <property name="text">
           <string>Manager</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">employeeTypeRadioButtons</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>12</height>
         </size>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="fireButton">
          <property name="text">
           <string>Fire</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hireButton">
          <property name="text">
           <string>Hire</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>CompanyWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="employeeTypeRadioButtons"/>
 </buttongroups>
</ui>
